config/environments/production.rb

```diff
@@ -66,7 +69,7 @@
   config.active_support.deprecation = :notify
 
   # For nginx:
-   config.action_dispatch.x_sendfile_header = 'X-Accel-Redirect'
+  config.action_dispatch.x_sendfile_header = 'X-Accel-Redirect'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.